pavelib/assets.py

```diff
@@ -172,14 +195,23 @@ def compile_templated_sass(systems, settings):
     """
     for sys in systems:
         sh(django_cmd(sys, settings, 'preprocess_assets'))
+        print("\t\tFinished preprocessing assets.")
 
 
-def process_xmodule_assets():
+def process_xmodule_assets(restart_django_server=False):
     """
     Process XModule static assets.
     """
     sh('xmodule_assets common/static/xmodule')
 
+    if restart_django_server:
+        # The python server needs to be reloaded (for some reason) in order to
```

Comment:
Yes, the static content for xmodules is hashed, so the file names change when the content changes. This was intended primarily as a way of deduplicating the content.